.infra/clickhouse-sync.yml

```diff
@@ -46,10 +46,13 @@ clickhouse.server.database: "%clickhouse_database%"
 database.allowPublicKeyRetrieval: "true"
 
 # snapshot.mode: Debezium can use different modes when it runs a snapshot. The snapshot mode is determined by the snapshot.mode configuration property.
-snapshot.mode: "initial"
+snapshot.mode: "never"
```

Comment:
This fixes the [issue](https://dailydotdev.slack.com/archives/C03C3JZPRU4/p1715239706260479)?

Reply:
No, original issue was caused by me manually editing yaml in k8s. Since you removed deployment, it's fixed now.